angular/projects/admin-nrpti/src/app/mines/mines-records-detail/mines-records-detail.component.html

```diff
@@ -0,0 +1,82 @@
+<main class="container-fluid-padding">
+  <section>
+    <div class="d-flex flex-column flex-sm-row justify-content-sm-between heading-bar">
+      <div>
+        <h1 id="name" class="m-0">{{ (record && record.recordName) || '-' }}</h1>
+      </div>
+    </div>
+  </section>
+
+  <section class="mb-3 pb-3 border-bottom">
+    <div class="row">
+      <div class="mb-2 col-12 col-sm-8">
+        <h2 class="border-0 mb-0">Record Information</h2>
+        <span class="grey-subtext">{{ lastEditedSubText }}</span>
+      </div>
+      <div class="col-12 col-sm-4">
+        <div class="d-flex flex-column">
+          <mat-slide-toggle
+            name="publishLng"
```

Comment:
Should probably rename/id the component `publishBcmi`, rather than LNG

Reply:
Fixed.